src/admin/components/Dialogs/StatusChangeModal.tsx

```diff
@@ -59,30 +59,43 @@ const StatusChangeModal = ({ handleClose, status, ...dialogProps }: StatusChange
   })();
 
   const dialogTitle = (() => {
-    let title = status === "approve" ? "Are you sure you want to approve this " : "Request more information for ";
+    let title = "";
 
+    switch (status) {
+      case "approve":
+        title = "Are you sure you want to approve this {}";
+        break;
+      case "moreinfo":
+        title = "Request more information for {}";
+        break;
+      case "restoration-in-progress":
+        title = "Are you sure you want to mark {} as Restoration In Progress?";
+        break;
+    }
+
+    let name;
     switch (resource as keyof typeof modules) {
       case "project":
-        title += record?.name ?? "Project";
+        name = record?.name ?? "Project";
         break;
       case "site":
-        title += record?.name ?? "Site";
+        name = record?.name ?? "Site";
         break;
       case "nursery":
-        title += record?.name ?? "Nursery";
+        name = record?.name ?? "Nursery";
         break;
       case "projectReport":
-        title += record?.title ?? "Project Report";
+        name = record?.title ?? "Project Report";
         break;
       case "siteReport":
-        title += record?.title ?? "Site Report";
+        name = record?.title ?? "Site Report";
         break;
       case "nurseryReport":
-        title += record?.title ?? "Nursery Report";
+        name = record?.title ?? "Nursery Report";
         break;
     }
 
-    return title;
+    return title.replace("{}", name);
```

Comment:
:art: This replace is a little weird. I'd do it instead by moving the `title` block down here and in each case of that block just do something like 
```
return `Request more information for ${name}`;
```